src/main/java/org/snomed/aag/rest/ServiceIntegrationController.java

```diff
@@ -79,8 +75,7 @@ public ServiceIntegrationController(CommitInformationValidator commitInformation
 						.body(message);
 			}
 
-			Branch sourceBranch = branchSecurityService.getBranchOrThrow(sourceBranchPath);
-			boolean pacComplete = projectAcceptanceCriteriaService.incrementIfComplete(projectAcceptanceCriteria, sourceBranch);
+			boolean pacComplete = projectAcceptanceCriteriaService.incrementIfComplete(projectAcceptanceCriteria, sourceBranchPath);
```

Comment:
Originally this method required a String, but was later changed to Branch. I have reverted to String.